templates/music/album_detail.html

```diff
@@ -4,6 +4,37 @@
 {% load markdown_deux_tags %}
 {% load music %}
 
+{% block socialmeta %}
+    <!-- Google+ data -->
+    <meta itemprop="name" content="{{ album.project.artist }} by {{ artist.name }}">
```

Comment:
I think you may have gotten confused by my earlier comment. I suggest you try rendering this page in your browser and inspecting the generated HTML. I think doing that you will have a better idea of what's being written here. This should probably say:
```html
<meta itemprop="name" content="{{ album.name }} by {{ album.project.artist.name }}">
```